lib/lambda_ethereum_consensus/fork_choice/helpers.ex

```diff
@@ -2,12 +2,35 @@ defmodule LambdaEthereumConsensus.ForkChoice.Helpers do
   @moduledoc """
     Utility functions for the fork choice.
   """
+  alias LambdaEthereumConsensus.ForkChoice
   alias LambdaEthereumConsensus.StateTransition.{Accessors, Misc}
+  alias LambdaEthereumConsensus.Store.{BlockStore, StateStore}
   alias SszTypes.BeaconBlock
   alias SszTypes.BeaconState
   alias SszTypes.Checkpoint
   alias SszTypes.Store
 
+  @spec current_status_message(Store.t()) ::
+          {:ok, SszTypes.StatusMessage.t()} | {:error, any}
+  def current_status_message(store) do
+    with {:ok, head_root} <- get_head(store),
+         {:ok, state} <- StateStore.get_state(head_root),
+         {:ok, signed_head_block} <-
+           BlockStore.get_block(head_root),
+         {:ok, finalized_checkpoint} <-
+           ForkChoice.Store.get_finalized_checkpoint() do
```

Comment:
you should be able to get all these from the `store` object, no need to call an external genserver